wrapper.ts

```diff
@@ -79,6 +79,8 @@ function setupMethods (soljson) {
       const copyFromCString = soljson.UTF8ToString || soljson.Pointer_stringify;
 
       const wrappedReadCallback = function (path: string, contents: string, error: string) {
```

Comment:
just run `npm run lint:fix` or the related package.json command to resolve these before committing.